lib/config/config-validator.js

```diff
@@ -112,7 +113,13 @@ function validateRuleOptions(rule, ruleId, options, source) {
             validateRuleSchema(rule, Array.isArray(options) ? options.slice(1) : []);
         }
     } catch (err) {
-        throw new Error(`${source}:\n\tConfiguration for rule "${ruleId}" is invalid:\n${err.message}`);
+        const enhancedMessage = `Configuration for rule "${ruleId}" is invalid:\n${err.message}`;
```

Comment:
Wondering if this should say "Inline configuration" (or equivalent), in order to make it clear that it's from a configuration comment rather than from a user's configuration files?

Reply:
I went with the current behavior on the assumption that it would be clear because the reported problem is at the comment's location.

I'd be fine with changing it if you feel strongly about it, although I wonder if we would want to refactor this function to make the error messages more flexible.